tests/test_action_loadconfig.py

```diff
@@ -0,0 +1,104 @@
+import mock
+import yaml
+
+from st2tests.base import BaseActionTestCase
+from loadconfig import NapalmLoadConfig
+
+from napalm.base import exceptions
+
+
+class NapalmLoadConfigTestCase(BaseActionTestCase):
```

Comment:
We should dd case for `config_file=False`, `config_text=None` maybe?

Reply:
Yes. I added a test-case that you mentioned in the last commit.